include/range/v3/utility/swap.hpp

```diff
@@ -29,6 +29,19 @@ namespace ranges
 {
     inline namespace v3
     {
+        /// \cond
+        namespace detail
+        {
+            template<typename T>
+            struct is_movable_
+              : meta::and_<
+                    std::is_object<T>,
+                    std::is_constructible<T, T>,
+                    std::is_assignable<T &, T>>
```

Comment:
They *are* equivalent, but e.g. `is_move_constructible<T>` is a bit higher level whereas `is_constructible<T, T>` requires analysis on the reader's part to figure out what it's doing.

But yeah, not a big deal either way.